src/swarm/neo/client/mixins/ClientCore.d

```diff
@@ -857,7 +855,24 @@ template ClientCore ( )
 
     public void abortAllRequests ( )
     {
-        // TODO
+        this.connections.request_set.abortAllRequests();
+    }
+
+    /***************************************************************************
+
+        Aborts a specific requests. This is useful if there's no way to
```

Comment:
Hmm, I'd be inclined to not support this until we have a really good need for it.

Reply:
Yes, it's a mistake that it's still open